packages/core-data/src/resolvers.js

```diff
@@ -500,6 +500,50 @@ export const __experimentalGetCurrentThemeGlobalStylesVariations =
 		);
 	};
 
+export const __experimentalGetCurrentThemeGlobalStylesRevisions =
```

Comment:
Missing doc comment.

Also probably doesn't need to be `__experimental`. In all likelihood it won't change and will end up as yet another item on our experimental API audit list for a dozen releases in a row.